core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -615,16 +615,18 @@ class Constants {
   }
 
   // ---- marker stuff ----
-  /** sentinel value indicating a marker follows */
-  readonly marker_sentinel = 0xFFFF;
+  /** == UC_SENTINEL, first item of a  */
+  readonly marker_sentinel     = 0xFFFF;
```

Comment:
should this be prefixed with `marker_` given it's a generic prefix which is currently only used with markers but may be used with other things in the future?

Reply:
maybe just uc_sentinel?

Reply:
> maybe just uc_sentinel?

yep sounds good, consistent with existing codebase, and not confusing in this context.